packages/fxa-content-server/app/scripts/lib/glean/index.ts

```diff
@@ -2,32 +2,90 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import Glean from '@mozilla/glean/web';
+
+import { accountsEvents } from './pings';
+import { name } from './event';
+import { uid } from './account';
+import { oauthClientId, service } from './relyingParty';
+import { deviceType, entrypoint, flowId } from './session';
+import * as utm from './utm';
+
 export type GleanMetricsConfig = {
   enabled: boolean;
   applicationId: string;
   uploadEnabled: boolean;
   appDisplayVersion: string;
   channel: string;
   serverEndpoint: string;
+  logPings: boolean;
+  debugViewTag: string;
 };
 
-import Glean from '@mozilla/glean/web';
-import * as registration from './registration';
+export type GleanMetricsContext = {
+  metrics: any;
+  relier: any;
+  user: any;
+  userAgent: any;
+};
+
+let gleanEnabled = false;
+let gleanMetricsConext;
+
+const populateProperties = () => {
+  const flowEventMetadata = gleanMetricsConext.metrics.getFlowEventMetadata();
+
+  // TODO when sending metrics for authenticated accounts
+  uid.set('');
```

Comment:
Just double checking this can be done later.

Reply:
Not sure if "can" as in feasible or okay to leave this empty for now, but yes to both.